sorc/ncep_post.fd/WETFRZLVL.f

```diff
@@ -120,10 +120,12 @@ SUBROUTINE WETFRZLVL(TWET,ZWET)
                     ZWET(I,J)=ZU
                   endif
                ENDIF
-               GOTO 20
+               EXIT loopL 
```

Comment:
This one should be "cycle loopi". So it will skip the code following the loopL just as "goto 20"

Reply:
@hu5970 If you look into the original one, the second "GOTO 20" is for jumping out of the "do 10 L" loop.